### PR TITLE
docs(masquerading): add missing "address"

### PR DESCRIPTION
**Pull request by raphink**
Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

### CODE REVIEW

File: Documentation/concepts/networking/masquerading.rst [hunk 1]
```diff
@@ -53,10 +53,10 @@ The current implementation depends on :ref:`the BPF NodePort feature <kubeproxy-
 The dependency will be removed in the future (:gh-issue:`13732`).
 
 Masquerading can take place only on those devices which run the eBPF masquerading
-program. This means that a packet sent from a pod to an outside will be masqueraded
-(to an output device IPv4 address), if the output device runs the program. If not
-specified, the program will be automatically attached to the devices selected by
-:ref:`the BPF NodePort device detection metchanism <Nodeport Devices>`.
+program. This means that a packet sent from a pod to an outside address will be
```
Comment (qmonnet) [inline Documentation/concepts/networking/masquerading.rst:56]: I'd have thought “to the outside...” instead, but it works too with “address”.

**Review by qmonnet — APPROVED** [review]
Looks good, thank you!

**Comment by glibsm** [comment]
This is still marked as backport-1.9, is that accurate?

**Comment by qmonnet** [comment]
No I think it no longer qualifies.

**Pull request merged**